jjb/dynamic/scale-ci_baseline.yml

```diff
@@ -137,6 +140,19 @@
         description: 'Duration to run'
         name: BASELINE_WORKLOAD_DURATION
         trim: 'false'
+    - string:
```

Comment:
This change will work when we are calling the individual job, the following minor changes are needed to get it working with the pipeline: the pipeline/build scripts read the properties file/config with the parameters and call the jobs. so the sample properties file ( https://github.com/openshift-scale/scale-ci-pipeline/blob/master/properties-files/baseline.properties ) and build script needs to be updated with the new parameters ( https://github.com/openshift-scale/scale-ci-pipeline/blob/master/pipeline-scripts/baseline.groovy#L46 ).

Reply:
makes sense @chaitanyaenr . I was looking at updating the individual tests as I'm not wanting to run them all. Would it be okay if I update the yaml, properties and groovy files for each test at a time and raise a pull request?

Reply:
Sounds good. Thanks.

Reply:
@chaitanyaenr - could you please review the most recent changes in this request?